source/include/TDCodecVia.h

```diff
@@ -257,7 +257,7 @@ void Format(SubString *format, Int32 count, StaticTypeAndData arguments[], Strin
 #define tsInputParamToken       "i"   // input parameter
 #define tsOutputParamToken      "o"   // output parameter
 #define tsInputOutputParamToken "io"  // input-output parameter
-                                      // (output and input-output params also act as dataItems, see above)
+                                      // (input, output, and input-output params also act as dataItems, see above)
```

Comment:
I think I didn't have input here intentionally, because you can't write to an input, so it could never be marked dirty.

Reply:
Commented below how this was a consistency change. And yea they can't be marked as dirty but they are "dataitems" in the loose sense of how the term is used which is roughly that they are coupled to controls / indicators. 

I can update the comment to that effect.

Reply:
Added a comment describing all params are dataitems